hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/hbase/TestHBaseIndex.java

```diff
@@ -263,6 +265,66 @@ public void testTagLocationAndDuplicateUpdate() throws Exception {
         && record.getCurrentLocation().getInstantTime().equals(newCommitTime))).distinct().count());
   }
 
+  @Test
+  public void testTagLocationAndPartitionPathUpdateWithRollback() throws Exception {
```

Comment:
so can you confirm that this test fails if not for the fix?

Reply:
yes